source/index-warning.txt

```diff
@@ -57,37 +57,32 @@ field is used in a query, but the field isn't indexed in the database:
       Filters.ne( "awards", "Comedy" )
    )
 
-The plugin shows this warning:
+The side panel shows the following warning under 
+:guilabel:`Performance Warnings`:
 
-.. code-block:: none
-   :copyable: false
-
-   This query will run without an index. If you plan on using this
-   query heavily in your application, you should create an index that
-   covers this query.
-   
-   Implement an Index
+.. image:: /images/index-warning.png
+   :alt: Index warning in the IntelliJ Plugin.
+   :width: 400px
 
 Create an Index
 ~~~~~~~~~~~~~~~
 
-To create an index for the query, click the :guilabel:`Implement an
-Index` link displayed with the warning in the plugin.
+To create an index for the query, click the :guilabel:`Create an
```

Comment:
(link observation) 
You have a link to the manual "Create an Index" on this page, above. It may be confusing to link out to the manual page with the same title as this section? Possibly consider updating to "For more information about indexes, see (https://www.mongodb.com/docs/manual/indexes/)"

Super happy to chat if this isn't clear.

Reply:
That makes sense! Updated.